dev-requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
 # This file defines the requirements.txt for development.
 #
-flake8==2.2.2
-pep8-naming==0.2.2
-nosier==1.1
+flake8
+pep8-naming
+nosier
```

Comment:
Doesn't it risk to break the app to remove pinned versions?

Reply:
That was needed for the upgrade, we have to keep in mind on pinning it again at the end of developments.